metricbeat/module/kubernetes/state_node/state_node.go

```diff
@@ -115,7 +115,17 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) {
 
 	m.enricher.Enrich(events)
 	for _, event := range events {
-		reporter.Event(mb.Event{MetricSetFields: event})
+		var namespace string
```

Comment:
Isn't the namespace for this metricset always `node`?

Should be the same value as `mapping.ExtraFields[mb.NamespaceKey]`